bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -318,7 +318,7 @@ data:
     {{- end }}
     printf "\nsentinel myid %s" "$(host_id "$HOSTNAME")" >> /opt/bitnami/redis-sentinel/etc/sentinel.conf
 
-    if [[ -z "$(getent ahosts "$HEADLESS_SERVICE" | grep -v "^${myip}")" ]]; then
+    if [[  ! $(get_sentinel_master_info) ]]; then
```

Comment:
I don't think you need the `[[` and `]]` in this case. Could you try removing it?

Reply:
sure, I will do that now